frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { UseOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+type Props = {
+    onSenderSelect: any; //What type should this be???
```

Comment:
Likely something like `(sender: string) => void` unless we want to explicitly type the Sender value.